cloc_multiple_directories/README.md

```diff
@@ -0,0 +1,24 @@
+# cloc_multiple_directories
```

Comment:
This could be the script's docstring. That way, we don't need a whole directory

Reply:
Done!